packages/starlight/vitest.workspace.ts

```diff
@@ -1 +1 @@
-export default ['__tests__/*'];
+export default ['__tests__/*/vitest.config.ts'];
```

Comment:
Since [Vitest 2.1.0](https://github.com/vitest-dev/vitest/releases/tag/v2.1.0), `*` matches anything, folders like before but also files which ends up being treated as a Vitest config so we explicitly matches against our explicit config files to ignore files at the root of the `__tests__/` folder.